examples/v1beta1/github/github-eventlistener-interceptor.yaml

```diff
@@ -15,13 +15,17 @@ spec:
                 secretName: github-secret
                 secretKey: secretToken
             - name: "eventTypes"
-              value: ["pull_request"]
-        - name: "only when PRs are opened"
-          ref:
-            name: "cel"
-          params:
-            - name: "filter"
-              value: "body.action in ['opened', 'synchronize', 'reopened']"
+              value: ["pull_request", "issue_comment"]
+            - name: "ownersCheckEnabled"
+              value: true
+            - name: "githubToken"
+              value:
+                secretName: github-token
+                secretKey: secretToken
+            - name: "enableOrgMemberCheck"
+              value: false
+            - name: "enableRepoMemberCheck"
+              value: false
```

Comment:
It would be great if you move this example to a different folder which focuses entirely for owner related stuff instead of changing in existing one

Reply:
agreed, moving into a different folder